### ISSUE TITLE
Accept `_` in number litterals

### ISSUE THREAD

**Issue by donnerc** [Tobias-Kohn/TigerPython-Parser#7]
The Python grammar allows underscores in the number litterals to increase readability. For instance, Python accepts litterals like `1_000_000` for 1 million. It would be great if the parser would accept that syntax.

**Comment by Tobias-Kohn** [comment]
You are right; this should have been included already a while back.  I will fix this as soon as possible.
